roles/libvirt_manager/README.md

```diff
@@ -63,6 +63,10 @@ cifmw_libvirt_manager_configuration:
       target: (Hypervisor hostname you want to deploy the family on. Optional)
       uefi: (boolean, toggle UEFI boot. Optional, defaults to false)
       bootmenu_enable: (string, toggle bootmenu. Optional, defaults to "no")
+      ip_address: (dict, having static IP address information. Optional)
+        address: (string, IP address to be assigned. Example 192.168.10.20/24. Optional)
+        gw: (string, gateway addressed to be configured. Required if address is set else optional.)
```

Comment:
else ignored more like?
can we do gateway assignment but not ip

Reply:
Thanks @marios 

Let me re-word it a bit. We do need it as we would be installing all required packages.

Reply:
To answer your question. Generally, you are allowed to set the IP address of  a system without specifying  a gateway address for the network via `ipv[46].gw` connection property in `nmcli`.